pkgs/applications/emulators/retroarch/cores.nix

```diff
@@ -727,6 +728,18 @@ in
     makefile = "Makefile";
   };
 
+  puae = mkLibRetroCore {
+    core = "puae";
+    description = "Amiga emulator based on WinUAE";
+    license = lib.licenses.gpl2Only;
+    makefile = "Makefile";
+    patches = fetchpatch {
+        url = "https://patch-diff.githubusercontent.com/raw/libretro/libretro-uae/pull/529.patch";
```

Comment:
Use the commit instead of the PR number, otherwise this can break in the future if the PR is updated.